### PR TITLE
chore: add ignore word

### PR DESCRIPTION
**Pull request by ming-tsai**
Hi @cclauss,
Check it out

### CODE REVIEW

**Comment by sridhar-5** [comment]
Hi @ming-tsai ,

Assign me for some reviews too please if possible.

**Comment by ming-tsai** [comment]
Hi @sridhar-5,
This change is for @cclauss on this issue he mentions it #178

**Comment by sridhar-5** [comment]
okay fine.

**Review by aayushjain7 — APPROVED** [review]

File: .github/workflows/lint_python.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
       - run: bandit --recursive --skip B101 .  # B101 is assert statements
       - run: black --check . || true
-      - run: codespell --ignore-words-list="ans,nnumber" --quiet-level=2  # --skip=""
+      - run: codespell --ignore-words-list="ans,nnumber,nin" --quiet-level=2  # --skip=""
```
Comment (cclauss) [inline .github/workflows/lint_python.yml:12]:
```suggestion
      - run: codespell --ignore-words-list="ans,nin,nnumber" --quiet-level=2  # --skip=""
```
Keeping the values sorted make it easy to find value and makes it difficult to add duplicate values.  This is easy when there are few values but gets more useful as lists get longer.

**Review by cclauss — APPROVED** [review]

**Pull request merged**